DependencyInjection/Configuration.php

```diff
@@ -103,6 +103,7 @@ public function getConfigTreeBuilder()
                     ->addDefaultsIfNotSet()
                     ->children()
                         ->scalarNode('default_format')->defaultNull()->end()
+                        ->scalarNode('method_prefix')->defaultTrue()->end()
```

Comment:
I just followed the same behavior as the existing option that appears in the doc http://symfony.com/doc/current/bundles/FOSRestBundle/7-manual-route-definition.html#method-name-prefix

But maybe we can add that option too